JYP-iOS/JYP-iOS/Sources/Presenter/NavigationBar/NavigationBarViewController.swift

```diff
@@ -0,0 +1,157 @@
+    func setNavigationBarTitleTextColor(_ color: UIColor?)
+    func setNavigationBarSubTitleText(_ text: String?)
+    func setNavigationBarSubTitleFont(_ font: UIFont?)
+    func setNavigationBarSubTitleTextColor(_ color: UIColor?)
+}
+
+class NavigationBarViewController: BaseViewController, BaseNavigationBarViewControllerProtocol {
+    // MARK: - UI Components
+    
+    var statusBar = UIView()
+    var navigaionBar = NavigationBar()
+    var contentView = UIView()
+    
+    // MARK: - Properties
+    
+    private let titleFont = JYPIOSFontFamily.Pretendard.medium.font(size: 16)
+    private let titleTextColor = JYPIOSAsset.textB75.color
+    private let subTitleFont = JYPIOSFontFamily.Pretendard.regular.font(size: 18)
+    private let subTitleTextColor = JYPIOSAsset.tagGrey200.color
+    
+    // MARK: - Life Cycle Methods
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        
+        setupNavigationBar()
+    }
+    
+    // MARK: - Setup Methods
+    
+    override func setupProperty() {
+        super.setupProperty()
+        
+        navigationController?.setNavigationBarHidden(true, animated: false)
+        setNavigationBarTitleFont(titleFont)
+        setNavigationBarTitleTextColor(titleTextColor)
+        setNavigationBarSubTitleFont(subTitleFont)
+        setNavigationBarSubTitleTextColor(subTitleTextColor)
+        navigaionBar.backButton.setImage(UIImage(systemName: "chevron.left"), for: .normal)
+    }
+    
+    override func setupHierarchy() {
+        super.setupHierarchy()
+        
+        view.addSubviews([statusBar, navigaionBar, contentView])
+      
+        navigaionBar.addSubviews([navigaionBar.title, navigaionBar.backButton, navigaionBar.subTitle])
+    }
+    
+    override func setupLayout() {
+        super.setupLayout()
+        
+        statusBar.snp.makeConstraints {
+            $0.top.leading.trailing.equalToSuperview()
+            $0.bottom.equalTo(view.safeAreaLayoutGuide.snp.top)
+        }
+        
+        navigaionBar.snp.makeConstraints {
+            $0.top.equalTo(statusBar.snp.bottom)
+            $0.leading.trailing.equalToSuperview()
+            $0.height.equalTo(60)
+        }
+        
+        navigaionBar.backButton.snp.makeConstraints {
+            $0.leading.equalToSuperview().inset(20)
+            $0.centerY.equalToSuperview()
+            $0.width.height.equalTo(24)
+        }
+        
+        navigaionBar.title.snp.makeConstraints {
+            $0.leading.equalToSuperview().offset(55)
+            $0.centerY.equalToSuperview()
+        }
+        
+        navigaionBar.subTitle.snp.makeConstraints {
+            $0.leading.equalTo(navigaionBar.title.snp.trailing).offset(12)
+            $0.centerY.equalToSuperview()
+        }
+        
+        contentView.snp.makeConstraints {
+            $0.top.equalTo(navigaionBar.snp.bottom)
+            $0.leading.trailing.bottom.equalTo(view.safeAreaLayoutGuide)
+        }
+    }
+    
```

Comment:
```suggestion

    override func setupBind() {
        navigaionBar.backButton.rx.tap
            .bind { [weak self] _ in
                self?.navigationController?.popViewController(animated: true)
            }
            .disposed(by: disposeBag)
    }


```

해당 클래스에서 backButton bind를 해주는 건 어떨까요?

Reply:
굿 아이디어 ,, 반영완료 했습니다 ~ !